core/templates/domain/collection/collection-node.model.spec.ts

```diff
@@ -16,12 +16,15 @@
  * @fileoverview Tests for collection-node.model.
  */
 
-import { CollectionNode } from
-  'domain/collection/collection-node.model';
+import { CollectionNode, CollectionNodeBackendDict}
+  from 'domain/collection/collection-node.model';
+import { LearnerExplorationSummaryBackendDict } from
+  'domain/summary/learner-exploration-summary.model';
 
 describe('Collection node model', () => {
   it('should provide an immutable exploration summary', () => {
-    var explorationSummaryBackendObject = {
+    var explorationSummaryBackendObject:
+    LearnerExplorationSummaryBackendDict = {
```

Comment:
```suggestion
      LearnerExplorationSummaryBackendDict = {
```

Reply:
@Nisarg-Chaudhari  fix the indentation at other places as well. thanks!

Reply:
Done!